src/Server.cpp

```diff
@@ -84,26 +84,32 @@ void Server::startPollEventLoop()
 void createFile(HTTPRequest &request)
 {
 	std::vector<File> files = request.getFiles();
-	std::string filename;
-	std::map<std::string, std::string>::iterator it = files.back().headers.find("filename");
+	std::vector<File>::iterator it;
 
-	if (it != files.back().headers.end())
-		filename = it->second;
-	else
+	// check each file
+	for (it = files.begin(); it != files.end(); ++it)
 	{
-		std::cout << "Error: file does not have a name" << std::endl;
-		return;
+		if (it->headers.find("filename") == it->headers.end())
+		{
+			std::cout << "Error: file does not have a name" << std::endl;
+			return ;
+		}
+
 	}
 
-	std::ofstream outfile(filename.c_str());
-	if (outfile.is_open())
+	// create files
+	for (it = files.begin(); it != files.end(); ++it)
 	{
-		outfile << files.back().fileContent;
-		outfile.close();
-		std::cout << "File created successfully" << std::endl;
+		std::ofstream outfile((it->headers.find("filename"))->second.c_str());
+		if (outfile.is_open())
+		{
+			outfile << it->fileContent;
+			outfile.close();
+			std::cout << "File created successfully" << std::endl;
```

Comment:
I think we should have a boolean for that, it's an improvement so it can be in a next PR